atomic_defi_design/qml/Wallet/EnableCoinModal.qml

```diff
@@ -125,8 +130,32 @@ BasicModal {
             text_value: qsTr("All assets are already enabled!")
         }
 
+        HorizontalLine {
+            Layout.fillWidth: true
+        }
+
+        DefaultRectangle {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 20
+            border.width: 0
+
+            DefaultText {
+                anchors.centerIn: parent
+                text: qsTr("You can still enable %1 assets. Selected: %2.")
+                        .arg(setting_modal.enableable_coins_count - API.app.portfolio_pg.portfolio_mdl.length)
+                        .arg(coin_cfg_model.checked_nb)
+            }
+        }
+
         // Buttons
         footer: [
+            DefaultButton {
+                property var enableable_coins_count: setting_modal.enableable_coins_count;
+                text: qsTr("Increase assets limit")
```

Comment:
Change this to "Change assets limit", as user might be decreasing it.

Reply:
implemented in https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/886/commits/2d32dcb85d16f8cf5ea3aece86a8710f49764646